### ISSUE TITLE
Weird problem with committing

### ISSUE THREAD

**Issue by DarkFaith** [zaggino/brackets-git#880]
**TL;DR; Commit button disabled despite having changes and untracked files. Previous commits pushed successfully. Deselecting and reselecting check-boxes does nothing. **

I added new images and modified an html page. The changes showed. I also had 2 JSON files I use for English and French translations that I had changed but not committed yet. I decided to commit those 2 files first so I use the check-boxes to select them and hit commit. My brackets became sluggish and the files remained there. 

The "commit" button was enabled so I clicked it a few times with intervals of around 10 seconds. I could still select and deselect the individual check-boxes for all the files (although the "select all" box wasn't altering the other check-boxes, just itself). I saw I had 2 un-pushed commits so I pushed them and it worked. The 2 files I had committed disappeared from the list (successfully committed, although I can't recall 100% if it was before or after the push). The "Commit" button was now disabled as well no matter what I selected (4 untracked files and 1 modified html page), clicking it did nothing.

I restarted brackets and the button remains disabled. Made more modifications to my html page and attempted to deselect and reselect the file check-boxes; nothing changed. I can still view the changelog on my html page and brackets is no longer sluggish after restarting it for the first time. Options to delete the untracked files and discard changes are still there. I uninstalled and reinstalled the extension with no luck. My working repo is on a USB key although it shouldn't affect anything. It was not pulled out or accidentally disconnected during this bug.

I'm thinking maybe git in general crashed, so I will reboot my laptop after submitting.
--------------------------------

*Brackets 1.1.0-15558 (win), Git 1.9.4.msysgit.2 & Brackets-Git 0.14.9*

**Comment by DarkFaith** [comment]
Rebooting has not solved the problem.

**Comment by zaggino** [comment]
@DarkFaith can you please check the logs from the console? (f12)
Ideally turn on the debug mode in the settings before doing so.

**Comment by DarkFaith** [comment]
Turned out to be pretty obvious. Had a index.lock file remaining from a git crash. Sorry for the bother and thanks a LOT for the help =)

**Comment by zaggino** [comment]
:+1:

**Comment by swizenfeld** [comment]
i am also getting locks on mac git version 2.1.2
